### PR TITLE
FIX > README, citation is fixed, authors.txt is added

### PR DESCRIPTION
**Pull request by denizdurduran**
This is a modificiation requested by Akiko and Karin

### CODE REVIEW

**Comment by denizdurduran** [comment]
With this PR, it also closes the Issue#1 for your note.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Optimum trajectory learning in musculoskeletal systems with model predictive control and deep reinforcement learning
 
-Berat Denizdurduran, Henry Markram, Marc-Oliver Gewaltig - Biol Cybern. 2022; 116(5-6): 711–726
+Denizdurduran, B., Markram, H., & Gewaltig, M.-O. (2022). Optimum trajectory learning in musculoskeletal systems with model predictive control and deep reinforcement learning. Biological Cybernetics, 116, 711–726. https://doi.org/10.1007/s00422-022-00940-x
+
+ 
+
+This is published open access under © The Authors CC BY 4.0 http://creativecommons.org/licenses/by/4.0/ which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made.
```
Comment (ASVG) [inline README.md:7]: Upon discussing with Karin, we request you to delete this sentence. It gets confusing if it is talking about the repo or the article you published.

Comment (ASVG) [inline README.md]: Upon discussing with Karin, we request you to delete this sentence. It gets confusing if it is talking about the repo or the article you published.

Comment (denizdurduran) [reply README.md]: done

**Pull request merged**